docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -10,27 +10,28 @@ This guide shows the various ways this can be done. For more of a background see
 
 ## Methods to pay a tx fee
 
-### Natively via an existing balance of fee juice
+1. Natively via an existing balance of fee juice
 
 Either your own account or someone else's address has a balance of fee juice, and the transaction is paid for from this balance.
 
-### Natively via a balance of fee juice to be claimed
+2. Natively via a balance of fee juice bridged from L1 to be claimed on L2
 
 Processing this transaction first claims bridged fee juice then is paid for from the balance, all in the same public/private tx.
 
-### Privately or publicly via a Fee Paying Contract
+3. Privately or publicly via a Fee Paying Contract
 
-A fee paying contract (FPC) is created and nominates a token that it accepts to then pay for txs in fee juice. So a user doesn't need to hold fee juice, they only need the token that the FPC accepts.
+A fee paying contract (FPC) is created and nominates a token that it accepts to then pay for transactions in fee juice. So a user doesn't need to hold fee juice, they only need the token that the FPC accepts.
 
-### Sponsored Fee Paying Contract (SponsoredFPC)
+4. Sponsored Fee Paying Contract (SponsoredFPC)
 
-The most straightforward way to pay for a transaction is via the sponsored fee payment method, bootstrapping some transactions by skipping the need to bridge fee juice to the account. This method uses a type of fee paying contract configured to pay for a number of transactions without requiring payment.
+The most straightforward way to pay for a transaction is via the sponsored fee payment method, bootstrapping some transactions by skipping the need to bridge fee juice to the account. This method uses a type of fee paying contract configured to pay for a number of transactions without requiring payment, but also requires that there is a sponsor to pay for the transactions.
 
-## Bridging fee-juice
+## Bridging Fee Juice
 
 To first get fee juice into an account it needs to be bridged from L1. You can skip this section if you want to first make a transaction via the SponsoredFPC.
```

Comment:
is there a section on minting fee juice already?

Reply:
I dont think so. are you wondering if this section is redundant?

Reply:
added a note on bridging when using testnet